english/coffee/coffee.md

```diff
@@ -14,6 +14,8 @@ After having your coffee do you want to:
 
 [Remenisce?](../remenisce/better-times.md).
 
+[Bake yourself something sweet to go with this delicious brew?](../cookie/bake-cookies.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.